### PR TITLE
Bug 1800459 - [RFE] Add column Storage Class for PVCs

### PR DESCRIPTION
**Pull request by cyril-ui-developer**
/assign @TheRealJon
/cc @benjaminapetersen @spadgett

### CODE REVIEW

File: frontend/public/components/persistent-volume-claim.jsx [hunk 1]
```diff
@@ -36,6 +36,7 @@ const tableColumnClasses = [
   classNames('col-lg-2', 'col-md-2', 'col-sm-4', 'hidden-xs'),
   classNames('col-lg-3', 'col-md-3', 'hidden-sm', 'hidden-xs'),
   classNames('col-lg-3', 'col-md-3', 'hidden-sm', 'hidden-xs'),
+  classNames('col-lg-3', 'col-md-3', 'hidden-sm', 'hidden-xs'),
```
Comment (spadgett) [inline frontend/public/components/persistent-volume-claim.jsx:39]: The column classes no longer add up to 12.

We should consider removing Bootstrap grid here as we've done in other places (for instance, https://github.com/openshift/console/blob/master/frontend/public/components/pod.tsx#L97-L108)

File: frontend/public/components/persistent-volume-claim.jsx [hunk 2]
```diff
@@ -116,7 +123,18 @@ const PVCTableRow = ({ obj, index, key, style }) => {
       <TableData className={tableColumnClasses[4]}>
         {_.get(obj, 'status.capacity.storage', '-')}
       </TableData>
-      <TableData className={tableColumnClasses[5]}>
+      <TableData className={classNames(tableColumnClasses[5])}>
+        {obj?.spec?.storageClassName ? (
```
Comment (spadgett) [inline frontend/public/components/persistent-volume-claim.jsx:127]: Optional chaining isn't necessary here. `obj.spec` will always be defined

File: frontend/public/components/persistent-volume-claim.jsx [hunk 3]
```diff
@@ -116,7 +123,18 @@ const PVCTableRow = ({ obj, index, key, style }) => {
...
+          <ResourceLink
+            kind="StorageClass"
+            name={obj?.spec?.storageClassName}
+            title={obj?.spec?.storageClassName}
```
Comment (spadgett) [inline frontend/public/components/persistent-volume-claim.jsx:131]: I'd leave off title.

File: frontend/public/components/persistent-volume-claim.jsx [hunk 4]
```diff
@@ -116,7 +123,18 @@ const PVCTableRow = ({ obj, index, key, style }) => {
...
+          />
+        ) : (
+          <div className="text-muted">-</div>
```
Comment (spadgett) [inline frontend/public/components/persistent-volume-claim.jsx:134]: Most places we don't use `text-muted` for `-`

File: frontend/public/components/persistent-volume-claim.jsx (hunk 2 above)
Comment (cyril-ui-developer) [reply frontend/public/components/persistent-volume-claim.jsx:127]: @spadgett This PR resolved optional chaining - https://github.com/openshift/console/pull/5382

File: frontend/public/components/persistent-volume-claim.jsx (hunk 1 above)
Comment (cyril-ui-developer) [reply frontend/public/components/persistent-volume-claim.jsx:39]: @spadgett There is another that resolved the col classes issue. - https://github.com/openshift/console/pull/4482

File: frontend/public/components/persistent-volume-claim.jsx (hunk 4 above)
Comment (cyril-ui-developer) [reply frontend/public/components/persistent-volume-claim.jsx:134]: @spadgett Will address this  and the removal of Bootstrap grid on a new PR.

File: frontend/public/components/persistent-volume-claim.jsx (hunk 1 above)
Comment (spadgett) [reply frontend/public/components/persistent-volume-claim.jsx:39]: @cyril-ui-developer Ack, thanks. I did just open a related bug

https://bugzilla.redhat.com/show_bug.cgi?id=1834361

**Pull request merged**